src/gltf-component-mappings.js

```diff
@@ -72,3 +72,22 @@ AFRAME.GLTFModelPlus.registerComponent("nav-mesh", "nav-mesh", (el, _componentNa
   // nav-mesh-helper will query for it later.
   el.setAttribute("nav-mesh");
 });
+
+AFRAME.GLTFModelPlus.registerComponent("pinnable", "pinnable");
+
+AFRAME.GLTFModelPlus.registerComponent("media", "media", (el, componentName, componentData) => {
+  if (componentData.id) {
+    el.setAttribute("networked", {
+      template: "#interactable-media",
+      owner: "scene",
+      lastOwnerTime: 1,
```

Comment:
`lastOwnerTime` is not actually a schema property of `networked` its just a value on it, so if you really want to do this you would need to do `el.components.networked.lastOwnerTime = 1`, though I suspect since it was working like this you don't actually need to set it at all.